docs/zkapps/advanced-snarkyjs.mdx

```diff
@@ -284,3 +284,121 @@ We imagine that one of the most important use cases for actions is to enable con
+
+```ts
+class BurnExample extends SmartContract {
+...
+@method burnTokens(addressToDecrease: PublicKey) {
+    this.experimental.token.burn({
+      address: addressToDecrease,
+      amount: 100_000,
+    });
+  }
+}
+```
+
+In the snippet above, we define a smart contract called `BurnExample` with a method called `burnTokens`. Similar to minting, we use the `this.experimental.token` property to call the `burn` method. This specifies the amount of tokens to burn for the specified address.
+
+:::note
+
+A zkApp cannot burn more tokens than the specified account has. If there is such a case, an error is thrown and no such transaction is made.
+
+:::
+
+##### Sending
+
+To send a custom token, use the `send` method available on `this.experimental.token`. See the example below of how a zkApp can authorize sending tokens between two accounts:
+
+```ts
+class SendExample extends SmartContract {
+...
+@method sendTokens(
+    senderAddress: PublicKey,
+    receiverAddress: PublicKey,
+    amount: UInt64
+  ) {
+    this.experimental.token.send({
+      to: receiverAddress,
+      from: senderAddress,
+      amount,
+    });
+  }
+}
+```
+
+In the snippet above, we define a smart contract called `SendExample` with a method called `sendTokens`. Then, in the same fashion, as minting and burning, we use the `this.experimental.token` property to call the `send` method.
+
+For a more comprehensive example of how to use custom tokens with a zkApp, please see this [custom token example](https://github.com/o1-labs/snarkyjs/blob/main/src/lib/token.test.ts)
+
+##### Understanding Important Terms
+
+If you are writing your zkApp to interact with custom tokens, you should understand the following essential terms:
+
+##### Token Id
+
+Token Ids are unique identifiers that are used to distinguish between different types of custom tokens. Custom token identifiers are globally unique across the entire network, meaning no two custom tokens can have the same token identifier.
+
+Token Ids themeselves are derived from a zkApp. To check the token id of a zkApp, you can use the `this.experimental.token.id` property.
+
+##### Token Accounts
+
+Token accounts are like regular accounts, but they hold a balance of a specific custom token instead of MINA. A token account is created from an existing account but is instead specified by a public key _and_ token id.
+If an existing account receives a transaction that is specified by a custom token, a token account for that public key and token id will be created if it does not exist.
+
+Token accounts are specific for each type of custom token, meaning that a single public key can have many different types of token accounts.
+
+To create a token account for a specified custom token, a token account is automatically created for that public key whenever an existing account receives a transaction denoted with a custom token.
```

Comment:
"To create a token account for a specified custom token, a token account is automatically created for that public key whenever an existing account receives a transaction denoted with a custom token." to "A token account is automatically created for a public key whenever an existing account receives a transaction denoted with a custom token."